modules/react/common/spec/focusRing.spec.ts

```diff
@@ -1,5 +1,6 @@
 import {memoizedFocusRing} from '../lib/styles/focusRing';
 
+// Check to delete this test since we removed memoized
```

Comment:
looks like it's unhappy about `.cache`

Reply:
does it still need that? `.cache` ?

Reply:
hey @mannycarrera4 it comes back to a question I asked on: https://github.com/Workday/canvas-kit/pull/2034/files#diff-fe5fd10f1d97d07137f3c450d5420b71a745c977270775b124a6201dd3d7ff0cR64. I am not entirely surely if focusRing needs to be memoized or not

Reply:
deleted entire spec as its not needed, it was only used to test memoization.